assets/js/index.js

```diff
@@ -0,0 +1,333 @@
+// global declarations
+const questions = [
+  {
+    question: "Which of these is not used to declare a variable in JavaScript?",
+    answers: ["Const", "Let", "Var", "Dec"],
+    correctAnswer: "Dec",
+  },
+  {
+    question: "What is the name for the symbols ( ) ?",
+    answers: [
+      "Curly Brackets",
+      "Angular Brackets",
+      "Parentheses",
+      "Open Brackets",
+    ],
+    correctAnswer: "Parentheses",
+  },
+  {
+    question:
+      "What attribute should be used to link a JavaScript file to an HTML file?",
+    answers: ["src", "href", "link", "rel"],
+    correctAnswer: "src",
+  },
+  {
+    question: "What symbol is used to reference an ID in CSS?",
+    answers: [".", "#", "/", "="],
+    correctAnswer: "#",
+  },
+  {
+    question: "Which of these is not a type of loop statement in JavaScript?",
+    answers: ["For", "While", "Rotate", "Continue"],
+    correctAnswer: "Rotate",
+  },
+];
+
+let questionIndex = 0;
+
+let timerValue = 12 * questions.length;
+
+let answerStatus = "not selected";
+
+let quizComplete = false;
+
+const startBtn = document.getElementById("start-button");
+
+const removeStart = document.getElementById("start-content");
+
+const mainContent = document.getElementById("main-content");
+
+// initialise local storage
+const onLoad = () => {
+  // check if highscores exists in LS
+  const highScores = readFromLocalStorage();
+  // if false then set highscores to empty array in LS
+  if (!highScores) {
+    localStorage.setItem("highscores", JSON.stringify([]));
+  }
+};
+
+// getScores from local storage
+const readFromLocalStorage = () => {
+  // get from LS by key
+  const getScores = localStorage.getItem("highscores");
+  // parse LS data
+  const parsedData = JSON.parse(getScores);
```

Comment:
We can directly return this line of code, rather than declaring it in a new variable just to return it. 

`return JSON.parse(getScores)`